app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -693,5 +698,21 @@ class BrowserTabViewModel(
     override fun externalAppLinkClicked(appLink: IntentType) {
         command.value = HandleExternalAppLink(appLink)
     }
+
+    override fun requiresAuthentication(request: BasicAuthenticationRequest) {
+        Timber.v("requiresAuthentication for [${request.site}]")
+        command.value = RequiresAuthentication(request)
+    }
+
+    override fun handleAuthentication(request: BasicAuthenticationRequest, credentials: BasicAuthenticationCredentials) {
+        request.handler.proceed(credentials.username, credentials.password)
+        // Attempt to save the credentials
+        command.value = SaveCredentials(request, credentials)
+    }
+
+    override fun cancelAuthentication(request: BasicAuthenticationRequest) {
+        Timber.v("cancelAuthentication, loading about:blank")
```

Comment:
i think we can kill this log statement; it's no longer accurate but i don't think it's needed generally